src/winforms/src/toga_winforms/widgets/canvas.py

```diff
@@ -316,5 +320,14 @@ def measure_text(self, text, font, tight=False):
             self._points_to_pixels(height),
         )
 
+    def as_image(self):
```

Comment:
The naming of this endpoint is a little misleading - it's returning image data, not an image.

We should either:
a. return an actual image from this API endpoint, or
b. rename this endpoint `get_image_data()`

I'm generally inclined to go with (b); the `get_` prefix would be consistent with the general theme of impl APIs being "get/set" driven (as they are all "interrogate a native layer to get/set the actual current state of the widget").

Reply:
How about `get_bitmap()`?

Because what is actually returning is a bitmap of the image.

Reply:
"bitmap" isn't entirely accurate either - it's bit data, but not necessarily a bitmap; it could also be PNG/JPG/etc format encoded bit data.